configure.ac

```diff
@@ -1204,13 +1207,15 @@ if [[ "$SUPPORTS_RDEVICE" = "yes" ]]; then
                  [Use IP network connection with the R: networking device (Linux/Unix/Win32) (default=ON)],
                  R_NETWORK,[Define to use IP network connection with the R: device.]
                 )
-        if [[ "$a8_host" != "win" ]]; then
+        if [[ "$a8_host" != "win" -a "$a8_host" != "macos" ]]; then
             A8_OPTION(rserial,yes,
                      [Use the host serial port with the R: networking device (Linux/Unix only) (default=ON)],
                      R_SERIAL,[Define to use the host serial port with the R: device.]
                     )
-        else
-            A8_NEED_LIB(ws2_32)
+        else 
+            if [[ "$a8_host" != "macos" ]]; then
```

Comment:
I would recommend to change this to `"$a8_host" = "win"` for clarity.

Reply:
agree.
changed.